core/src/main/scala-3/caliban/schema/SchemaDerivation.scala

```diff
@@ -29,7 +29,11 @@ trait SchemaDerivation[A] {
    * GraphQL needs 2 different types, and they can't have the same name.
    * By default, we add the "Input" suffix after the type name.
```

Comment:
Can you also edit the comment to specify it's not done if the suffix is already present?

Reply:
Did my best, but it will not win any nobel literature prize, feedback wanted 🙏